pkg/apis/iam/v0alpha1/types_servier_account.go

```diff
@@ -24,3 +24,19 @@ type ServiceAccountList struct {
 
 	Items []ServiceAccount `json:"items,omitempty"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type ServiceAccountTokenList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+
+	Items []ServiceAccountToken `json:"items,omitempty"`
+}
+
+type ServiceAccountToken struct {
```

Comment:
Should we add `metav1.ObjectMeta` to sub resources?

Reply:
Unless it would be populated reasonably, i do not think so